### DISCUSSION TITLE
Where was the first modern ⛳🥅🏋️‍♀️🚴‍♂️ held?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#896] [Galaxy Trivia] [answered]
**Bonus:** How is it different from the ancient games?

**Comment by peterbaccaro** [comment, marked as answer]
The first modern Olympic Games were held in **Athens, Greece**. The games took place from April 6 to April 15, 1896.

The modern Olympic Games and the ancient Olympic Games had several differences:

- **Origin**: The ancient Olympic Games originated in ancient Greece around the 8th century BCE, while the modern Olympic Games were revived in the late 19th century.

- **Participation**: The ancient Olympic Games were limited to male athletes from the city-states of ancient Greece, while the modern Olympic Games are open to participants from all countries and both genders.

- **Sports**: The ancient Olympic Games consisted primarily of athletic events such as running, discus, javelin, and chariot races. In contrast, the modern Olympic Games include a wide range of sports, including athletics, swimming, gymnastics, cycling, soccer, basketball, and many more.

- **Religious Significance**: The ancient Olympic Games were deeply intertwined with religious beliefs and rituals, honouring the Greek god Zeus. The modern Olympic Games have a more secular nature, focusing on athletic competition and international goodwill.

- **Duration**: The ancient Olympic Games lasted for several days, usually up to a week, while the modern Olympic Games are held over a period of about two weeks.

- **Venue**: The ancient Olympic Games were held in Olympia, Greece, at a specific site dedicated to the games. In contrast, the modern Olympic Games are held in different host cities around the world, with the venue changing every four years.

- **Professionalism**: The ancient Olympic Games were exclusively for amateur athletes, and participation was restricted to those who were not engaged in any form of paid work or trade. In contrast, the modern Olympic Games allow both amateur and professional athletes to compete.

While the modern Olympic Games draw inspiration from the ancient Olympics, they have evolved significantly to reflect the changing times, expanded sports, and a more global approach to participation.
